Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/AcpiPlatform.c

```diff
@@ -1053,6 +1056,49 @@ PlatformUpdateAcpiGnvs (
     PlatformNvs->PcieSlot1PowerEnableGpio = GPIO_VER4_S_GPP_E1;
     PlatformNvs->PcieSlot2RpNumber = 5;
     break;
+  case PLATFORM_ID_ADL_P_DDR5_RVP:
```

Comment:
Is this code snippet only applicable for DDR5 flavor? How about LPDDR ?

Reply:
This need to be verified for other flavors, but lets do that after RC1 release